### ISSUE TITLE
Textfield component does not honour aria-describedBy when getErrorMessage is also provided as a prop

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#4068]
### Bug Report

- __Package version(s)__: 5.46.1005
- __Browser and OS versions__: Chrom

#### Describe the issue:

Scenario 1: aria-describedBy, Working fine
We are passing aria-describedby to text field component which it renders correctly and links the <input> to that description Id.

Scenario 2: aria-describedBy with getErrorMessage or error message, major issue
When we pass getError message/ error message as a prop along with aria-describedby Id, then the component never honors the id that the consumer has passed both in error or non error state.
In non-error state, the component should honor the id passed by the consumer. In error mode, the component should honor the internal description id (check this._descriptionId in code).

#### Expected behavior:

The component should always honor the consumer props. It should bring in its own intelligence when in error state. But when not in error state, aria-describedby passed by the consumer should be honored.

**Comment by bennettclark** [comment]
We recommend using the **Textfield** component's `description` prop rather than the native `aria-describedby` prop, because `description` automatically creates the description span with unique ID for you.

I've submitted a Pull Request to add support for the `aria-describedby` native prop as a fallback if the `description` and `errorMessage` props are not set.
